internal/provider/nordvpn/updater.go

```diff
@@ -0,0 +1,26 @@
+package nordvpn
```

Comment:
Move this back to `internal/provider/nordvpn/updater/updater.go`
Maybe this is due to a bad rebase (see files deleted internal/provider/nordvpn/updater/*.go)